pkg/web/templates/partials/transaction/transaction_list.html

```diff
@@ -17,54 +18,61 @@
       {{ if .Transactions }}
       {{ range .Transactions }}
       <tr>
+        <td class="cursor-pointer text-center" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">
+          {{ if eq .Source "local" }}
+          <i class="fa-solid fa-file-export" title="Outgoing transaction from local VASP"></i>
+          {{ else if eq .Source "remote" }}
+          <i class="fa-solid fa-file-import" title="Incoming transaction from remote VASP"></i>
+          {{ else }}
+          <i class="fa-solid fa-question" title="Unknown transaction type"></i>
+          {{ end }}
+        </td>
         {{ if .Status }}
-        <div class="flex items-center gap-x-1">
-          {{ if eq .Status "Draft" }}
+          {{ if eq .Status "draft" }}
           <td class="cursor-pointer" onclick="transaction_modal.showModal()" hx-get="/v1/transactions/{{ .ID }}" hx-target="#transaction_modal" hx-swap="innerHTML">
             <div class="flex items-center gap-x-1">
               <img src="/static/draftStatusIcon.svg" alt="" />
               <span>{{ .Status }}</span>
```

Comment:
I've realized the text color for the status must have been removed during a previous merge conflict. I'll create a story to fix as I'll have to find a previous PR to remember what the values should be.